packages/plugin-browser-session/session.js

```diff
@@ -30,8 +31,15 @@ const sessionDelegate = {
           user: sessionClient._user
         }
       ]
-    })
+    }
+
+    const ignore = runSyncCallbacks(sessionClient._cbs.sp.slice(0), payload, 'onSessionPayload', sessionClient._logger)
```

Comment:
Don't think `sessionClient._cbs.sp.slice(0)` is technically needed as `runSyncCallbacks` also copies the array with slice.

Reply:
`slice(0)` could also be replaced with `slice()` although neither change affects the bundle size AFAICT

Reply:
Good point, though I can't remember why the array is copied at all 🤔

Reply:
the sync callback runner copies the array and pops of functions until there are none left – that's why it needs to be copied.